course_catalog/task_helpers.py

```diff
@@ -605,3 +611,81 @@ def parse_bootcamp_json_data(course_data, force_overwrite=False):
             )
             course.prices.add(course_price)
         index_func(course)
+
+
+def parse_bootcamp_json__data_bootcamp(bootcamp_data, force_overwrite=False):
```

Comment:
Could you fix the name to avoid the double underscore?

Reply:
Fixed